src/forms/FormFieldAddress/AddressDrawer/AddressDrawer.tsx

```diff
@@ -293,8 +294,7 @@ const AddressDrawer = (props: AddressDrawerProps): ReactElement => {
 			console.warn('Country response from google "' + addressComponentsMap.country.label + '" could not be found in the list of countries provided in getOptionsCountries.');
 			componentsNotFound += `${componentsToAddress.country}, ${componentsToAddress.administrative_area_level_1}, `;
 		}
-
-		await setFieldValue("address1", addressComponentsMap.route.label, true);
+		await setFieldValue("address1", `${addressComponentsMap.street_number.label} ${addressComponentsMap.route.label}`, true);

```

Comment:
This assumes that street_number will always be present, so whenever the user adds an address that doesn't have a number it results in an empty space 
![image](https://user-images.githubusercontent.com/23409284/224164945-10f12594-e8f0-433b-84d9-54f3f2e282e1.png)